docs/install/SetupPrereqs.md

```diff
@@ -20,8 +20,23 @@ Percona Everest has two primary components:
 
 ### Supported k8s clusters
 
-* Google Kubernetes Engine (GKE) (1.24 - 1.28)
-* Amazon Elastic Kubernetes Service (EKS) (1.24 - 1.28)
+Percona Everest works on most of the cloud K8s and on most of the on-prem vanilla K8s. 
+
+However, not all the many combinations of K8s distributions and K8s versions might be fully tested and certified. Please refer to the matrix below, and feel free to [contact us](SetupPrereqs.md#get-expert-help) in case of questions.
+
+| Platform              | Kubernetes Version | Percona Everest Version | State                                   | 
+|:----------------------|:-------------------|:------------------------|:----------------------------------------| 
+| Google GKE            | 1.24 - 1.28        | >= 1.0.0                | Fully tested & certified                |
+| Amazon EKS            | 1.24 - 1.28        | >= 1.0.0                | Fully tested & certified                |
+| Vanilla K8s (kubeadm) | 1.24 - 1.28        | >= 1.0.0                | Fully tested & certified                |
+| Azure AKS             | -                  | >= 1.0.0                | Works but not fully certified yet       |
+| DigitalOcean          | 1.29 - 1.30        | >= 1.0.0                | Works but not fully certified yet       |
```

Comment:
Where did these versions (1.29-1.30) come from? The ones for GKE/EKS come from our DB operators supported versions but why is this one different?

Reply:
Hi!  Thanks for your comment

- As far as I see from https://docs.digitalocean.com/products/kubernetes/details/supported-releases/
  - 1.24, .25 .26 .27 are EOL.  .28 will be EOL soon
  - 1.29 and 1.30 are the two stable with EOL in 2025 

I put 1.29 and 1.30 based on previous internal discussions in the team channel.
Hope this helps

Reply:
Since our QA team didn't test this platform I think we should do the same we did for AKS, i.e. say `Works but not fully certified yet` but don't have any specific versions.

Reply:
Thanks. Ok from my side. @rasika-chivate can we also add Peter as reviewer, please - for general feedback and because I know he gave a test to these Digital Ocean versions. So would like to hear his opinion as well. Final decision is on Peter/Diogo obiously. Thanks!

Reply:
Peter is on PTO until next week so he won't be able to review this before the release.

I know that Peter tests on digital ocean but he doesn't run a full test suite to make sure everything works as expected so you were right by saying `Works but not fully certified yet`.
In general, I think that we can only mention specific k8s versions when we say `Fully tested & certified`. Otherwise, it can be misleading.

We are now working on automating a lot of the manual tests that QAs currently do, hopefully in the near term it will be much easier to expand all of these platforms to a fully certified status.

For now, we should abstain from advertising specific versions for digital ocean.

Reply:
done. thanks again. resolving